packages/discovery/src/discovery/provider/AllProviders.ts

```diff
@@ -69,13 +69,23 @@ export class AllProviders {
         })
       }
 
+      const celestiaApiClient = new CelestiaApiClient({
```

Comment:
It's kinda stupid that it's required. I'd want it to be optional like beaconApiUrl above.